crates/bevy_render/Cargo.toml

```diff
@@ -49,7 +49,7 @@ parking_lot = "0.11.0"
 bevy-glsl-to-spirv = "0.1.7"
 
 [target.'cfg(target_os = "ios")'.dependencies]
-shaderc = "0.6.2"
+shaderc = { git = "https://github.com/google/shaderc-rs", branch = "master"}
```

Comment:
@cart I remember reading that you didn't want git dependencies because it blocks crates.io releases, so does that mean we can't land this?

Reply:
this has a crate anyways why not just switch it to the crate https://crates.io/crates/shaderc

Reply:
@simlay landed a commit on `shaderc-rs` master a few days ago to fix the build for iOS. We would need a new release or a `tmp_cart_shaderc` kinda thing until the next release comes out.

Reply:
Yeah I have no problem publishing tmp versions of crates if we need to, but we should ask the shaderc-rs folks if they're willing to cut a release first. If there isn't a new release by the time we drop `bevy 0.3`, we can commit to using a tmp crate. I'm down to use a tmp crate in master in the short term as well, just so we can get this merged